spec/unit/views/components/table_for_spec.rb

```diff
@@ -301,6 +301,46 @@
       end
     end
 
+    context 'when i18n option is specified' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.